pages.es/common/amass-track.md

```diff
@@ -0,0 +1,12 @@
+# amass track
+
+> Seguimiento de las diferencias entre enumeraciones del mismo dominio.
+> Más información: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-track-subcommand>.
+
+- Muestra la diferencia entre las dos últimas enumeraciones del dominio especificado:
+
+`amass track -dir {{ruta/a/directorio_de_base_de_datos}} -d {{nombre_dominio}} -last 2`
```

Comment:
```suggestion
- Show the difference between the last enumerations of a specific domain::

`amass track -dir {{ruta/a/directorio_de_base_de_datos}} -d {{nombre_dominio}} -last {{1..infinity}}`
```

Optional fix: `Show the difference between the last enumerations of a specific domain:` can be used as a translation source, `specific` may sound better.
Mandatory fix: `-last` option doesn't accept just `2` integer literal, placeholder is required, range syntax can be used.

Reply:
Based on the above suggestion feel free to translate it.

Reply:
Done. :)